doc-site/.vitepress/theme/components/PageHeader.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <div class="header-container">
+            <h1 class="title">{{title}}</h1>
+        <img :src=getImage(imagePath) alt="">
+    </div>
+  </template>
+  
+  <script setup lang="ts">
+    import { useData } from 'vitepress'
+  type Props = {
+    title: string,
+    imagePath: string,
+    pageLevel: string
+  }
+  const { title, imagePath, pageLevel } = defineProps<Props>();
+
+  const { isDark } = useData()
+
+  const getImage = (imagePath: string) => {
+    let pageLevelString = "../"; 
+    switch (pageLevel) {
+      case "2":
+      pageLevelString = "../../"
+      case "3":
+      pageLevelString = "../../../"
+    }
+    switch (imagePath) {
+      case "intro":
+        return isDark.value ? pageLevelString +'assets/images/intro-dark-mode.svg':pageLevelString+ 'assets/images/intro.png';
```

Comment:
jeg ser at bildene ikke bundles og de finnes ikke etter appen er bygget. Grunnen er fordi her importerer vi bildene egentlig ikke og siden vite-press lager statiske sider, den skjønner ikke at den må bundle disse bildene uten en spesifisert import. Forresten jeg tror ikke vi trenger pageLevel property her. Jeg klarte å få bildene etter bygg på plass med 
```js
<script setup lang="ts">
import { useData } from 'vitepress';
import introDarkMode from '../../../assets/images/intro-dark-mode.svg';
import intro from '../../../assets/images/intro.png';
import designServicesDarkMode from '../../../assets/images/design-services-dark-mode.svg';
import designServices from '../../../assets/images/design-services.png';
import codeBlocksDarkMode from '../../../assets/images/code-blocks-dark-mode.svg';
import codeBlocks from '../../../assets/images/code-blocks.png';

type Props = {
  title: string;
  imagePath: string;
  pageLevel: string;
};
const { title, imagePath, pageLevel } = defineProps<Props>();

const { isDark } = useData();

const getImage = (imagePath: string) => {
  switch (imagePath) {
    case 'intro':
      return isDark.value ? introDarkMode : intro;
    case 'designer':
      return isDark.value ? designServicesDarkMode : designServices;
    case 'developer':
      return isDark.value ? codeBlocksDarkMode : codeBlocks;
  }
};
</script>
```
vi trenger ikke å bekymre oss om at bildene importeres for mange ganger. Under bygg de bundles en gang uansett

Reply:
Snyggt, mye bedre!